File: 63.verificar_primo/verificar_primo.py
# Ejercicio No. 63: Verificar número primo

# processing
def es_primo(n):
    for i in range(2, n):
        if n % i == 0:
            return False
    return True

# input
n = int(input("Cantidad de números: "))

# processing
i = 2
s = "Primos: "
np = 0
while np < n:
    if es_primo(i):
        s = s + str(i) + ","
        np = np + 1
    i = i + 1

# output
print(s)

File: 63.verificar_primo/VerificarPrimo.java
package VerificarNumeroPrimo;

import javax.swing.JOptionPane;

public class VerificarPrimo {
    //Processing
    public static boolean esPrimo(int n){
        boolean primo = true;
        for(int i = 2; i<n; i++){
            if(n%i == 0){
                primo = false;
            }
        }
        return primo;
    }
    
    public static void main(String[] args) {
        //Input
        int n = Integer.parseInt(JOptionPane.showInputDialog("Cantidad de números:"));

        //Processing
        int i = 2;
        String s = "Primos: ";
        int np = 0;

        while(np < n){
            if(esPrimo(i)){
                s = s + i + ",";
                np ++;
            }
            i++;
        }

        //Output
        JOptionPane.showMessageDialog(null, s);
    }
}
